_commands/basics/true.md

```diff
@@ -0,0 +1,38 @@
+---
+---
+
+true
+--
+
+`true` is a command that does nothing and return a true value(0). 
```

Comment:
If you change ...
~~~markdown
`true` is a command that does nothing and return a true value(0).
~~~
... to ...
~~~markdown
`true` is a command that returns the logical value of truth, which is `0`.
~~~
... then I think that takes care of everything for me.

If no one else has any comments, then I'll merge you in. Thank you so much!